docs/user/managing-devices.md

```diff
@@ -417,6 +453,53 @@ spec:
 [...]
 ```
 
+## Scheduling Updates and Downloads
+
+Flight Control supports time-based scheduling for update and download operations using cron style expressions. This allows you to restrict system modifications to defined maintenance windows or operational periods.
+
+Each device can define two independent schedules in the `updatePolicy` section of the `DeviceSpec`:
+
+- **`downloadSchedule`**: Defines when the device is allowed to download update artifacts such as OS image layers.
+- **`updateSchedule`**: Defines when the device is allowed to apply updates.
+
+Each schedule supports:
+
+- **`at`**: A [cron expression](https://man7.org/linux/man-pages/man5/crontab.5.html) (`"0 2 * * *"`) specifying valid run times.
+- **`timeZone`**: (Optional) The time zone in which the schedule is evaluated. Defaults to the device’s local system time zone. Must be a valid [IANA time zone](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones).
+- **`startGraceDuration`**: (Optional) A duration string like `"30m"` that extends the allowed start time window after a schedule trigger.  
+  Follows the [Go duration format](https://pkg.go.dev/time#ParseDuration), such as `"1h"` or `"45m"`.
+
+Flight Control evaluates these schedules during its control loop to determine whether each policy is currently allowed to proceed.
```

Comment:
```suggestion
The Flight Control agent evaluates these schedules during its control loop to determine whether each policy is currently allowed to proceed.
```

Reply:
Should we document which update state the device remains in while waiting for the scheduled time and/or link to `device-api-statuses.md#device-update-status`?